## Description: Install external scripts in FHS friendly location.
## Origin/Author: Mitchell Smith <mjs@mjsprojects.net>
Index: xwax-0.9/Makefile
===================================================================
--- xwax-0.9.orig/Makefile	2011-04-16 19:19:28.000000000 +1000
+++ xwax-0.9/Makefile	2011-04-21 08:32:59.315641837 +1000
@@ -35,7 +35,7 @@
 # Installation paths
 
 BINDIR = $(PREFIX)/bin
-EXECDIR = $(PREFIX)/libexec
+EXECDIR = $(PREFIX)/share/xwax
 MANDIR = $(PREFIX)/share/man
 DOCDIR = $(PREFIX)/share/doc
 
